home/components/HomeScreenHeader.tsx

```diff
@@ -0,0 +1,80 @@
+import { spacing } from '@expo/styleguide-native';
+import { Button, View, Row, Image, UserIcon, scale, Text } from 'expo-dev-client-components';
+import * as Haptics from 'expo-haptics';
+import { CurrentUserDataFragment } from 'graphql/types';
+import * as React from 'react';
+import { Platform } from 'react-native';
+
+import { useTheme } from '../utils/useTheme';
+
+type Props = {
+  currentUser?: CurrentUserDataFragment;
+};
+
+export function HomeScreenHeader({ currentUser }: Props) {
+  const { theme, themeType } = useTheme();
+
+  return (
+    <Row
+      padding="medium"
+      align="center"
+      bg="default"
+      style={{
+        justifyContent: 'space-between',
+      }}>
+      <Row align="center">
+        <View
+          align="centered"
+          rounded="medium"
+          shadow="button"
+          style={{
+            backgroundColor: theme.background.default,
+            height: scale.xl,
+            width: scale.xl,
+            marginRight: spacing[2],
```

Comment:
these 4 should be available as props for `View`

Reply:
actually maybe not margin right but the others

Reply:
im thinking of adding "darkModeStyle" "lightModeStyle" props to all these primitives to make it easier in the markup - wdyt?

Reply:
`style.backgroundColor` gets overriden by the `bg` prop when used together

Reply:
im a bit confused - you have `bg=default` and are also adding the `backgroundColor` style to be `theme.default`? shouldn't they be the same thing?

Reply:
For light theme, i want default, but for dark, I want overlay. I think I'm gonna push a commit to this branch that also adds the overlay bg property so that this doesn't have to be done via custom styles